html-folder/alerts.html

```diff
@@ -98,14 +100,14 @@ <h1 class="text-shadow"> PISTON <span class="highlight-orange-text">UI</span></h
         </div>
         <div class="navigation-end-part">
 
-            <a href="https://github.com/Dev-AkshatGupta" class="nav-btn-icon highlight-white-text" target="blank"><i
+            <a href="https://github.com/Dev-AkshatGupta" class="nav-btn-icon text-black" target="blank"><i
                     class="fab fa-github"></i>
             </a>
-            <a href="https://twitter.com/Akshat86604233" class="nav-btn-icon highlight-white-text" target="blank"><i
+            <a href="https://twitter.com/Akshat86604233" class="nav-btn-icon text-black" target="blank"><i
                     class="fab fa-twitter"></i>
             </a>
             <a href="https://www.linkedin.com/in/akshat-gupta-257914157/" class="nav-btn-icon
-        highlight-white-text" target="blank"><i class="fab fa-linkedin-in"></i>
+    text-black" target="blank"><i class="fab fa-linkedin-in"></i>
```

Comment:
target should be  _blank please change that

Reply:
both _blank and blank work so no worries for that